### PR TITLE
add: Ext PG integration to Flink docs

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes

Add content for ext PG integration with Aiven for Apache Flink
New topics: 
 - Integrate Aiven for Apache Flink® with external PostgreSQL
 - Create a PostgreSQL® CDC or JDBC connector-based Apache Flink®

Vale fixes to other related articles. 

[HH-3427](https://aiven.atlassian.net/browse/HH-3427)

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

[HH-3427]: https://aiven.atlassian.net/browse/HH-3427?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+   integrated service.
+1. In the **Table SQL**, enter the SQL statement for your chosen connector:
+
+   - For a **CDC connector**:
+
+     ```sql
+      CREATE TABLE external_pg_cdc_source (
+        id INT PRIMARY KEY,
+        data VARCHAR
+      ) WITH (
```
Comment (gpkc) [inline docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md:47]: From my experimenting, it was necessary to add a `slot.name` parameter as well, e.g.:
`'slot.name' = 'my_slot_name'`

**Review by gpkc — APPROVED** [review]

File: docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md:47]: Thanks. Added this.

**Review by gpkc — APPROVED** [review]

File: docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+1. Go to the **Aiven for Apache Flink** service page and
+   click **Application** in the sidebar.
+1. Click **Create new application**, enter a name your application, and
+   click **Create application**.
+
+   :::note
+    If modifying an existing application, create a new version for source or s
+    ink table changes.
+   :::
```
Comment (ArthurFlag) [inline docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md:33]: I'm not sure I understand this note, what do we mean? also the `s <new link> ink` seems off 🙂

File: docs/products/flink/howto/ext-pg-flink-integration.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  avn service integration-list --project <project-name> <flink-service-name>
+```
+
+Parameters:
+- `--project <project-name>`: The name of your Aiven project.
+- `<flink-service-name>`: The name of your Aiven for Apache Flink service.
+
+To create Aiven for Apache Flink applications, you will need the
+integration ID of the Aiven for Apache Flink service. Obtain the
+`integration_id` from the integration list.
```
Comment (ArthurFlag) [inline docs/products/flink/howto/ext-pg-flink-integration.md:92]: Let's skip the future to shorten things? 

```suggestion
To create Aiven for Apache Flink applications, retrieve
the integration IDof the Aiven for Apache Flink
service from the integration list.
```

File: docs/products/flink/howto/ext-pg-flink-integration.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+### Create Aiven for Apache Flink applications
+
+With the integration ID obtained, you're now ready to develop Aiven for Apache Flink
+applications that utilize the data from your external PostgreSQL database
+as either a source or a sink.
```
Comment (ArthurFlag) [inline docs/products/flink/howto/ext-pg-flink-integration.md:98]: We've already mentioned this at the top of the page and previous section, let's limit repetition?

```suggestion
```

File: docs/products/flink/howto/ext-pg-flink-integration.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+Integrating your Aiven for Apache Flink service with an external PostgreSQL
+database can be achieved through the Aiven Console by following these steps:
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and choose your project.
+1. From the **Services page**, you can either
+   - Create a new [**Aiven for Apache Flink service](/docs/platform/howto/create_new_service)
+   - Select an existing service
+1. Configure an external PostgreSQL database integration endpoint:
+   - Go to the **Projects** screen where all services are listed.
```
Comment (ArthurFlag) [inline docs/products/flink/howto/ext-pg-flink-integration.md:112]: Is this the **Services** sidebar entry?

File: docs/products/flink/howto/ext-pg-flink-integration.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+## Configure integration using Aiven Console
+
+Integrating your Aiven for Apache Flink service with an external PostgreSQL
+database can be achieved through the Aiven Console by following these steps:
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and choose your project.
+1. From the **Services page**, you can either
+   - Create a new [**Aiven for Apache Flink service](/docs/platform/howto/create_new_service)
+   - Select an existing service
```
Comment (ArthurFlag) [inline docs/products/flink/howto/ext-pg-flink-integration.md:110]: I think all the substeps of this whole list are all sequential?

**Review by ArthurFlag — CHANGES_REQUESTED** [review]
Minor comments added 🙂

File: docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md (hunk 2 above)
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/ext-pg-cdc-jdbc-connector.md]: Update text for clarity.

File: docs/products/flink/howto/ext-pg-flink-integration.md (hunk 3 above)
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/ext-pg-flink-integration.md]: Move this line to the next section and updated the content.

**Comment by harshini-rangaswamy** [comment]
Currently on hold. Will reopen depending on the Flink status.

**Pull request closed without merging**